uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -56,7 +56,12 @@ <h4 class="modal-title" ng-switch="documentModal.isRole('admin')">
       </ui-select>
     </div>
   </div>
-</div>
+    <div class="col-xs-1">
```

Comment:
Please add a class to align, same as above.

Reply:
Can you expatiate on this. Should i move the tool icon closer to the drop menu?

Reply:
I meant the `control-label` class that aligns it vertically.